misc/kaprekars

```diff
@@ -0,0 +1,105 @@
+/*
+Ioannis Sykas Leriotis 29/09/2019
+6174 is the Kaprekar Constant. This number is special as we always get this number when following steps are followed 
+for any four digit number such that all digits of number are not same.
+
+Sort four digits in ascending order and store result in a number “asc”.
+Sort four digits in descending order and store result in a number “desc”.
+Subtract number larger number from smaller number, i.e., abs(asc – desc).
+Repeat above three steps until the result of subtraction doesn’t become equal to the previous number.
+*/
+
+# include <stdio.h>
+# include <stdbool.h>
+# include <math.h>
+
+int kaprekarRec(int n, int prev)
```

Comment:
what is the need to pass  **prev** as arguments if you are assigning it a different value in the function _(Line 23)_ ?